pyvo/mivot/utils/json_encoder.py

```diff
@@ -0,0 +1,30 @@
+import json
+import numpy
+
+
+class JsonEncoder(json.JSONEncoder):
+    """
```

Comment:
I don't know it, but Numpy suggests to use custom encoders: (https://numpy.org/devdocs/user/how-to-io.html). Am I a step behind?

Reply:
Class `JSONEncoder` renamed as `MivotJSONEncoder`. The name is more appropriate because the encoder is used to build JSON from MIVOT annotations, not from Numpy data.